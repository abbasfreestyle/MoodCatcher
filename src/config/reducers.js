import { combineReducers } from 'redux';

import { moodName, moodReducer } from 'modules/Mood';

export const reducers = {
  [moodName]: moodReducer
};

const combinedReducers = combineReducers(reducers);

export default combinedReducers;
