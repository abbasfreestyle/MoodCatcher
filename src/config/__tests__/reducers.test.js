import { reducers } from '../reducers';

describe('Reducers.', () => {
  describe('Snapshots.', () => {
    test('should match', () => {
      expect(reducers).toMatchSnapshot();
    });
  });
});
