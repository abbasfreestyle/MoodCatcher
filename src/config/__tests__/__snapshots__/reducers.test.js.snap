// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Reducers. Snapshots. should match 1`] = `
Object {
  "Mood": [Function],
}
`;
